.github/workflows/ci-nightly.yml

```diff
@@ -26,7 +26,8 @@ jobs:
       run: |
         git config user.name ${{ env.GITHUB_PUBLISH_USER }}
         git config user.email ${{ env.GITHUB_PUBLISH_EMAIL }}
-        git remote set-url origin https://${{GITHUB_SERVICE_PAT}}:${{env.GITHUB_SERVICE_PAT}}@github.com/microsoft/fast.git
```

Comment:
Can you explain this change and how it relates to Code Security Scanning or why the change is needed?

Reply:
This has nothing to do with Code security scanning. This workflow is for deployment. It is failing on env secrets value.  Since now continuous deployment is not in place, it's not a big concern. We will fix this next when all the testing work is finished.